tools/run_python_system_tests.py

```diff
@@ -32,8 +32,8 @@
 
 drivers_using_other_driver = ['niscope', 'nifgen', 'nidigital', 'nitclk', ]
 if args.module in drivers_using_other_driver:
-    # Creating the wheel for the other required module only uses Python 3.9
-    command += ['-e', 'py310-{0}-wheel_dep,'.format(args.module)]
+    # Creating the wheel for the other required module only uses Python 3.11
```

Comment:
This is the one part I'm not sure of. It seems this was added with 3.8 as the version back when 3.8 was the latest version supported. I think I recall us having some kind of tox issues with earlier versions of Python. Maybe this was intended to be run for 3.8 and later? I'm not quite sure.

Actually, the closer I look, the less certain I am that this file is even used.

Reply:
`drivers_using_other_driver` is for things like `niscope` using `nitclk`.
@texasaggie97-zz  this looks ok, right?

Reply:
Since this file doesn't appear to be used, I have created #1918 to delete it.

Reply:
All the checks for the other PR passed, so I will mark this resolved with the assumption that the other PR will go in.